lib/ansible/modules/git.py

```diff
@@ -43,23 +43,28 @@
         default: "HEAD"
     accept_hostkey:
         description:
-            - If C(yes), ensure that "-o StrictHostKeyChecking=no" is
-              present as an ssh option.
+            - Will ensure or not that "-o StrictHostKeyChecking=no" is present as an ssh option.
+            - Be aware that this disables a protection against MITM attacks.
+            - Those using OpenSSH >= 7.5 might want to set I(ssh_opt) to 'StrictHostKeyChecking=accept-new'
+              instead, it does not remove the MITM issue but it does restrict it to the first attempt.
         type: bool
         default: 'no'
         version_added: "1.5"
     ssh_opts:
         description:
-            - Creates a wrapper script and exports the path as GIT_SSH
-              which git then automatically uses to override ssh arguments.
-              An example value could be "-o StrictHostKeyChecking=no"
-              (although this particular option is better set by
-              I(accept_hostkey)).
+            - Options git will pass to ssh when used as protocol, it works via GIT_SSH_OPTIONS.
+            - For older versions it appends GIT_SSH_OPTIONS to GIT_SSH/GIT_SSH_COMMAND.
+            - Other options can add to this list, like I(key_file) and I(accept_hostkey).
+            - An example value could be "-o StrictHostKeyChecking=no" (although this particular
```

Comment:
The example could use a different option than one already covered above as you wrote , like `Compression=yes`

Reply:
kept original as it stresses how other options can also modify the opts.